app/views/homes/index.html.erb

```diff
@@ -0,0 +1,9 @@
+ <div class="container shadow mt-5" id="main-page">
+  <h1 class="text-center fw-bold">SmartSpend Tracker</h1>
+  <div class="fixed-bottom">
+    <div class="d-grid gap-2">
+      <%= link_to 'LOG IN', new_user_session_path, class:"btn btn-primary" %>
+      <%= link_to 'SIGN UP',new_user_registration_path, class:"btn btn-white border" %>
+    </div>
+  </div>
+</div>
```

Comment:
- [ **OPTIONAL** ] As required for the project and to make your design looks smarter, I think it would be better if you improve the design of the home page to be close to the one from the **[template](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)**. I suggest you center the project title

| Template design | Your Design |
| ---------------------- | ------------------- |
| ![ksnip_20231121-171135](https://github.com/zbahati/budgetApp/assets/83042742/6c69126d-2704-47a0-8897-c6858a7e8781) | ![Screenshot from 2023-11-23 15-06-29](https://github.com/zbahati/budgetApp/assets/83042742/d6499fbd-0034-42f5-8074-82def25c814c) |